FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift

```diff
@@ -74,7 +74,10 @@ public final class ThreadMessageSenderCellNode: CellNode {
         automaticallyManagesSubnodes = true
 
         senderNode.attributedText = input.sender
+        senderNode.accessibilityIdentifier = "senderEmail"
         dateNode.attributedText = input.date
+        senderNode.accessibilityIdentifier = "date"
+
```

Comment:
here you meant `dateNode.acc..`

Reply:
no, 
`senderNode.accessibilityIdentifier = "senderEmail"`

Reply:
Please carefully re-read the code I quoted.

Reply:
yes, sorry, fixed already, 
will push